lms/djangoapps/commerce/apps.py

```diff
@@ -0,0 +1,17 @@
+"""
+Commerce Application Configuration
+"""
+from django.apps import AppConfig
+
+
+class CommerceConfig(AppConfig):
+    """
+    Application Configuration for Commerce.
+    """
+    name = 'commerce'
+
+    def ready(self):
+        """
+        Connect handlers to signals.
+        """
+        from . import signals  # pylint: disable=unused-variable
```

Comment:
It looks like there's still an import of `signals` in `__init__.py` that should get removed.

Reply:
Good catch to you both - I missed that. It's removed now.